AK/Atomic.h

```diff
@@ -112,6 +112,13 @@ static inline V* atomic_load(T volatile** var, MemoryOrder order = memory_order_
     return __atomic_load_n(const_cast<V**>(var), order);
 }
 
+static inline void atomic_pause()
+{
+#if __has_builtin(__builtin_ia32_pause)
+    __builtin_ia32_pause();
+#endif
+}
```

Comment:
Should this also have `__builtin_arm_yield()` or something like that?

Reply:
Yep, nice catch! That builtin is not documented, but it totally works!